Jenkinsfile

```diff
@@ -141,7 +141,7 @@ def startBraveBrowserBuild() {
                                 credentials('brave-builds-github-token-for-pr-builder')
                                 github('brave/devops', 'https')
                             }
-                            branch('master')
+                            branch('gn_check')
```

Comment:
this needs revert

Reply:
@mihaiplesa - will revert once the PR is ready to merge.